source/carla_host.py

```diff
@@ -1003,6 +1011,18 @@ def slot_pluginAdd(self):
         if not self.host.add_plugin(btype, ptype, filename, None, label, uniqueId, extraPtr, 0x0):
             CustomMessageBox(self, QMessageBox.Critical, self.tr("Error"), self.tr("Failed to load plugin"), self.host.get_last_error(), QMessageBox.Ok, QMessageBox.Ok)
 
+    @pyqtSlot()
+    def slot_confirmRemoveAll(self):
+        if self.fPluginCount == 0:
+            return
+
+        if QMessageBox.question(self, self.tr("Remove All"),
+                                      self.tr("Are you sure you want to remove all plugins?"),
+                                      QMessageBox.Yes|QMessageBox.No) == QMessageBox.No:
+                                        return
+
+        self.slot_pluginRemoveAll()
+
     @pyqtSlot()
     def slot_pluginRemoveAll(self):
```

Comment:
Since this function is no longer being used as a connection slot, better to remove the decoration and prefix from it.